zookeeper-docs/src/main/resources/markdown/zookeeperCLI.md

```diff
@@ -57,7 +57,7 @@ ZooKeeper -server host:port cmd args
 	reconfig [-s] [-v version] [[-file path] | [-members serverID=host:port1:port2;port3[,...]*]] | [-add serverId=host:port1:port2;port3[,...]]* [-remove serverId[,...]*]
 	redo cmdno
 	removewatches path [-c|-d|-a] [-l]
-	set [-s] [-v version] path data
+	set [-s] [-v version] path data [-f input_file_name]
```

Comment:
shouldn't it be 
```suggestion
	set [-s] [-v version] path [-d data] [-f input_file_name]
```
? what does it mean when both data and input file are provided?

Reply:
data and -f both options together are not valid, added check and test to cover that.